examples/Basic.hs

```diff
@@ -22,18 +22,26 @@ mean xs = sum' xs / fromIntegral (length xs)
 
 main :: IO ()
 main = do
-    handle <- forkServer "localhost" 8000
-    counter <- getCounter "iterations" handle
-    label <- getLabel "args" handle
-    event <- getDistribution "runtime" handle
+    handle <- forkServer "0.0.0.0" 8000
+    baseCounter  <- getCounter "iterations" [] handle
+    fizzCounter  <- getCounter "iterations" ["dim0:fizz"] handle
```

Comment:
Regarding the comment I gave on the pull request in ekg-core. I think its here I'd rather see that the dimension names don't have to be repeated (albeit that does imply they need to be positional).